src/modules/75-cf/pages/environment-details/__tests__/EnvironmentDetailsBody.test.tsx

```diff
@@ -74,56 +69,35 @@ const existingKey: ApiKey = {
 }
 
 describe('EnvironmentDetailsBody', () => {
-  const addNewAPIKey = (newKey: ApiKey): void => {
-    mutateMock.mockResolvedValue({ ...newKey })
-
-    userEvent.click(screen.getByRole('button', { name: /addKey/ }))
-    userEvent.type(screen.getByRole('textbox'), newKey.name)
-
-    useGetAllAPIKeysMock.mockReturnValue(getMockResponseData([{ ...newKey }]))
-
-    userEvent.click(screen.getByRole('button', { name: /create/ }))
-  }
-
   beforeAll(() => {
     const { getComputedStyle } = window
     window.getComputedStyle = elt => getComputedStyle(elt)
   })
 
   beforeEach(() => {
     jest.clearAllMocks()
-    useGetAllAPIKeysMock.mockReturnValue(getMockResponseData())
   })
 
-  test('it should display the copy button and redaction warning when new API key is created', async () => {
-    const newKey: ApiKey = {
-      identifier: 'NewKey',
-      apiKey: 'api-key',
-      name: 'NewKey',
-      type: 'server'
+  test('it should render correct empty state', async () => {
+    const hasNoKeys = {
+      itemCount: 0,
+      pageCount: 0,
+      pageIndex: 0,
+      pageSize: 0,
+      apiKeys: []
     }
+    renderComponent({ data: hasNoKeys })
```

Comment:
```suggestion
      renderComponent({data: {itemCount: 0,
      pageCount: 0,
      pageIndex: 0,
      pageSize: 0,
      apiKeys: [] }})
```
Same here with the const thing if you want